docs/user-guide.rst

```diff
@@ -20,23 +20,26 @@ First things first, import the urllib3 module:
 
 .. code-block:: python
 
-    >>> import urllib3
+    import urllib3
 
 You'll need a :class:`~poolmanager.PoolManager` instance to make requests.
 This object handles all of the details of connection pooling and thread safety
 so that you don't have to:
 
 .. code-block:: python
 
-    >>> http = urllib3.PoolManager()
+    http = urllib3.PoolManager()
 
 To make a request use :meth:`~poolmanager.PoolManager.request`:
 
 .. code-block:: python
 
-    >>> r = http.request('GET', 'http://httpbin.org/robots.txt')
-    >>> r.data
-    b'User-agent: *\nDisallow: /deny\n'
+    import urllib3
+    # Send the GET request.
+    r = urllib3.request('GET', 'https://httpbin.org/robots.txt')
```

Comment:
May as well use proper names like `resp` instead of `r` if we're modifying the example

Reply:
Yeah, did that in the latest commit